modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddObjectDefinition.tsx

```diff
@@ -187,6 +233,16 @@ type TInitialValues = {
 	label: string;
 	name?: string;
 	pluralLabel: string;
+	storageType: string;
+};
+
+type ObjectDefinition = {
+	label: {[key: string]: string};
+	name?: string;
+	objectFields: unknown;
```

Comment:
Is the `objectFields` really unknown? It seems to me the we could say it is an array at least
```suggestion
	objectFields?: unknown[];
```